docs/changelog/102492.yaml

```
@@ -0,0 +1,5 @@
+pr: 102492
+summary: Respect `ChunkedDataExtractor's` query in the wrapped `ScrollDataExtractor`
```

The release notes should be phrased in a way that end users could understand.

```suggestion
summary: Ensure datafeed previews with no start or end time don't search the cold or frozen tiers
```

---

Thanks, I'll take care of that myself next time.